README.md

```diff
@@ -14,6 +14,11 @@ This repository is a Drupal client module that can connect to a contentpool
 server. You'll need a drupal project for installing it. Refer to "Installation"
 for details.
 
+## Dependencies
+
+For the contentpool filter selection, there is a dependency on Vue 2.5.17 which
```

Comment:
It's bad practice to depend on specific versions without reason. We should say we depend on Vue 2.5 or later -> semantic versioning to the rescue. Anyway, minor.